content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -236,8 +236,9 @@ The following configuration variables are available **only** when using automati
 
 `DD_TRACE_ENABLED`
 : **TracerSettings property**: `TraceEnabled`<br>
-Enables or disables all automatic instrumentation. Setting the environment variable to `false` completely disables the CLR profiler. For other configuration methods, the CLR profiler is still loaded, but traces will not be generated. Valid values are: `true` or `false`.<br>
+Enables or disables all instrumentation. Valid values are: `true` or `false`.<br>
 **Default**: `true`
+**Note**: Setting the environment variable to `false` completely disables the client library, and it cannot be enabled through other configuration methods, such as configuration files or in code. If it is set to `false` through another configuration method (not an environment variable), the client library is still loaded, but traces will not be generated.
```

Comment:
Suggest removing examples, makes it sound like there might be another way to enable this that's not listed.
```suggestion
**Note**: Setting the environment variable to `false` completely disables the client library, and it cannot be enabled through other configuration methods. If it is set to `false` through another configuration method (not an environment variable), the client library is still loaded, but traces will not be generated.
```